docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,354 @@
+## Istio Ambient mode
+
+This documentation is specific to the Istio Ambient mode using Sail Opeartor installation. These features provide early access to upcoming product features, enabling users to test functionality and provide feedback during the development process.
+
+## Introduction to Istio Ambient mode
+
+Ambient mesh and its reference implementation with Istio's ambient mode was announced in September 2022. It reaches a stable status in Istio 1.24 release. The core innovation behind ambient mesh is that it slices Layer 4(L4) and Layer 7(L7) processing into two distinct layers. Istio's ambient mode is powered by lightweight, shared L4 node proxies and optional L7 proxies, removing the needfor traditional sidecar proxies from the data plane.
+
+The lightweight shared L4 node proxy is called the `ztunnel`(zero-trust tunnel). ztunnel drastically reduces the overhead of running a mesh by removing the need to potentially over-provision memory and CPU within a cluster to handle expected loads, while still providing security using mutual TLS with cryptographic identity, simple L4 authorization policies, and telemetry.
+
+The L7 proxies are called `waypoints`. Waypoints process L7 functions such as traffic routing, authorization policy enforcement, and resilience. Waypoints run outside of your application deployments and can scale independently based on your needs.
+
+In contrast to the earlier injection of sidecars. Users can start with the secure L4 overlay, which offers features such as mTLS, authorization policy, and telemetry. Complex L7 handling such as retries, traffic splitting, load balancing, and observability collection can then be enabled on a case-by-case basis.
+
+## Component version
+
+For running Istio ambient mode on OpenShift, we recommend using Sail Operator and installing `Istio`, `IstioCNI` and `ZTunnel` resources version v1.24.0 or newer.
+
+## Concepts
+
+### ZTunnel resource
+
+The `ZTunnel` resource is used to manage your L4 node proxy. It is a cluster-wide resource as it will install a `DaemonSet` that will be operating on all nodes of your cluster. You can select a version by setting the `spec.version` field, as you can see in the sample below. Just like the `Istio` resource, it also has a `values` field that exposes all of the options provided in the `ztunnel` chart:
+
+```yaml
+apiVersion: sailoperator.io/v1alpha1
+kind: ZTunnel
+metadata:
+  name: default
+spec:
+  profile: ambient
+  version: v1.24.0
+  namespace: ztunnel
+  values:
+    ztunnel:
+      image: docker.io/istio/ztunnel:1.24.0
```

Comment:
Instead of using `image` in the example (which is matching with the version), shall we use logging?
I'm fine, if you prefer to retain the image example if its a more widely used use-case.

```
values:
  logging:
    level: "all:debug"
```

Reply:
ok, I will keep the image example there because we may switch to other image definition in the future.